frontend/src/extensions/extra-integrations/blockquote/__snapshots__/blockquote-extra-tag-markdown-extension.test.tsx.snap

```diff
@@ -0,0 +1,181 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`blockquote extra tag renders the tag "> [color=#f00] text" correctly 1`] = `
+<div>
+  <blockquote
+    style="border-left-color: #f00;"
+  >
+    
+
+    <p>
+       text
+    </p>
+    
+
+  </blockquote>
+  
+
+</div>
+`;
+
+exports[`blockquote extra tag renders the tag "[=value]" correctly 1`] = `
+<div>
+  <p>
+    [=value]
+  </p>
+  
+
+</div>
+`;
+
+exports[`blockquote extra tag renders the tag "[color=#abcdef]" correctly 1`] = `
+<div>
+  <p>
+    <span
+      class="blockquote-extra"
+      style="color: rgb(171, 205, 239);"
+    >
+      <i
+        class="fa  fa-tag mx-1 "
+      />
+    </span>
+  </p>
+  
+
+</div>
+`;
+
+exports[`blockquote extra tag renders the tag "[color=#dfe]" correctly 1`] = `
+<div>
+  <p>
+    <span
+      class="blockquote-extra"
+      style="color: rgb(221, 255, 238);"
+    >
+      <i
+        class="fa  fa-tag mx-1 "
+      />
+    </span>
+  </p>
+  
+
+</div>
+`;
+
+exports[`blockquote extra tag renders the tag "[color=]" correctly 1`] = `
+<div>
+  <p>
+    [color=]
+  </p>
+  
+
+</div>
+`;
+
+exports[`blockquote extra tag renders the tag "[color=blauge]" correctly 1`] = `
+<div>
+  <p>
+    <span
+      class="blockquote-extra"
+    >
+      <i
+        class="fa  fa-tag mx-1 "
+      />
+      blauge
```

Comment:
Is this right?

Reply:
Yes because it's not a real color

Reply:
The word is some kind of inside joke. I changed it to something else.